persons.rb

```diff
@@ -0,0 +1,27 @@
+class Person
+  # creating accessors
+  attr_accessor :name, :age, :id
```

Comment:
- [ ] @lmunozm1702 kindly ensure only name and age have setter and getter, the id should only be a getter as in the project description
![image](https://user-images.githubusercontent.com/61408860/215509609-9498074e-d6a9-40ca-9808-aeaf245ab1c5.png)